common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -98,6 +99,14 @@
 _ = lambda text: text
 
 
+# TODO: remove this before the merge
```

Comment:
These are just for making sure the video tests run fine in presence of waffle in the module, I'll remove them before merge, you may ignore the changes in this file.